globals.d.ts

```diff
@@ -0,0 +1,71 @@
+/// <reference types="node" />
+/// <reference types="react" />
```

Comment:
can you remove commented code?

Reply:
They're not comments but instructions for the ts compiler, I didn't know that too 😄